chatbot/start-bot.py

```diff
@@ -1,183 +1,150 @@
-from langchain import LLMChain, PromptTemplate
-from langchain.memory import ConversationBufferWindowMemory
-from langchain.callbacks.base import AsyncCallbackHandler
-
+from langchain.tools.python.tool import PythonREPLTool
+from langchain.python import PythonREPL
+from langchain.prompts.prompt import PromptTemplate
+from langchain.prompts import MessagesPlaceholder
+from langchain.memory import ConversationBufferMemory
+from langchain.schema.messages import (
+    SystemMessage,)
+from langchain.schema.messages import AIMessage, HumanMessage
+from langchain.callbacks.base import BaseCallbackHandler
 
-# load dotenv
-from dotenv import load_dotenv
-load_dotenv()
 
 SERVER_URL = "https://ai.imjoy.io"
-
-test_story = """
-
-<iframe src="https://imjoy.io" style="width:100%;height:300px"></iframe>
-
-Once upon a time in the charming town of Storyville, there lived a highly curious cat named Whiskers. Whiskers was no ordinary cat; he had a peculiar knack for finding lost items. From misplaced keys to the forgotten old relics of the town, Whiskers found them all. His reputation as the finder of lost things brought him much affection from the townsfolk, but also an unending chain of requests.
-
-One day, Whiskers was approached by a worried little girl named Sophie. With tears in her eyes, she told Whiskers that she had lost her grandmother's locket while playing in the park. This locket was no ordinary piece of jewelry; it contained a picture of her late grandmother, making it invaluable to Sophie.
-
-Moved by her distress, Whiskers set off on the search for the locket. He combed through every inch of the park, scampered up the trees, and investigated every bush. Hours turned into days, but the locket was nowhere to be found. Whiskers felt despair creeping in, something he had never experienced before.
-
-Instead of giving up, Whiskers decided to change his approach. He started asking the other animals in the town for help. The birds offered to provide an aerial view, the squirrels scoured the treetops, and even the usually aloof town dog offered to sniff around.
-
-Together, they launched a search operation, each contributing in their own way. After a long day of exhaustive search, it was the tiny town mouse who found the locket tucked away inside a small hole at the base of an old tree. 
-
-Whiskers returned the precious locket to Sophie, who was overjoyed. The whole town celebrated their success, demonstrating the power of unity and perseverance. From that day on, Whiskers was not just known as the finder of lost things but also as a beloved community hero who knew the power of asking for help and working together. This adventure was a heartwarming testament to the fact that sometimes, even the smallest creatures could make the biggest difference.
+token = login({"server_url": SERVER_URL})
+server = connect_to_server(
+    {"name": "hypha bot", "server_url": SERVER_URL, "token": token, "method_timeout": 60, "sync_max_workers": 1}
```

Comment:
```suggestion
    {"name": "hypha bot", "server_url": SERVER_URL, "token": token, "method_timeout": 60, "sync_max_workers": 2, "webrtc": True}
```

Let's set sync_max_workers and enable webrtc.